packages/js/src/Modules/Verto/webrtc/BaseCall.ts

```diff
@@ -138,10 +138,34 @@ export default abstract class BaseCall implements IWebRTCCall {
     this._targetNodeId = what;
   }
 
+  /**
+   * Gets the local stream of the call.
+   * This can be used in a video/audio element to play the local media.
+   * [MediaStream](https://developer.mozilla.org/en-US/docs/Web/API/MediaStream).
+   *
+   * @examples
+   *
+   * ```js
+   * const stream = call.localStream();
+   * <video id="localId" srcObject={stream} autoPlay='autoplay' controls={false} />
```

Comment:
Actually, this is not valid JS. You could do it like:
```suggestion
   * document.querySelector('video').srcObject = stream;
```